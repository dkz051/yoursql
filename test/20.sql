CREATE DATABASE OOP;
USE OOP;
CREATE TABLE qwe(id INT, aint INT, bchar CHAR, cdouble DOUBLE, PRIMARY KEY(id));

INSERT INTO qwe(id, aint, bchar, cdouble) VALUES( 1,      11, 'a',     100.0);
INSERT INTO qwe(id, aint, bchar         ) VALUES( 2, 4144959, 'b'           );
INSERT INTO qwe(id, aint, bchar, cdouble) VALUES( 3,      13, 'c',     300.0);
INSERT INTO qwe(id, aint,        cdouble) VALUES( 4,      14,      4144959.0);
INSERT INTO qwe(id, aint, bchar, cdouble) VALUES( 5,      15, 'e',     500.0);
INSERT INTO qwe(id, aint, bchar, cdouble) VALUES( 6, 4144959, 'f', 4144959.0);
INSERT INTO qwe(id, aint, bchar, cdouble) VALUES( 7,      17, 'g',     700.0);
INSERT INTO qwe(id,       bchar, cdouble) VALUES( 8,          'h', 4144959.0);
INSERT INTO qwe(id, aint, bchar, cdouble) VALUES( 9,      19, 'i',     900.0);
INSERT INTO qwe(id, aint, bchar, cdouble) VALUES(10,      20, 'j', 4144959.0);

SELECT * FROM qwe;
SELECT * FROM qwe WHERE aint > 50;
SELECT * FROM qwe WHERE cdouble > 1000.0;
SELECT bchar FROM qwe;
SELECT * FROM qwe WHERE aint = 4144959;

DROP DATABASE OOP;
