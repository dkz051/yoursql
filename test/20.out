id	aint	bchar	cdouble
1	11	a	100
2	4144959	b	NULL
3	13	c	300
4	14	NULL	4144959
5	15	e	500
6	4144959	f	4144959
7	17	g	700
8	NULL	h	4144959
9	19	i	900
10	20	j	4144959
id	aint	bchar	cdouble
2	4144959	b	NULL
6	4144959	f	4144959
id	aint	bchar	cdouble
4	14	NULL	4144959
6	4144959	f	4144959
8	NULL	h	4144959
10	20	j	4144959
bchar
a
b
c
NULL
e
f
g
h
i
j
id	aint	bchar	cdouble
2	4144959	b	NULL
6	4144959	f	4144959
